src/Page/Definition.elm

```diff
@@ -84,23 +86,38 @@ renderExpandableCategories model categories =
         (\listPosition ->
             -- We maybe want to ditch this div for valid html - but this is simple
             div []
-                [ renderTerm (categoryKeysFromListPosition listPosition)
+                [ renderTerm model (categoryKeysFromListPosition listPosition)
                 , renderDefinition model (categoryKeysFromListPosition listPosition)
                 ]
         )
         categories
 
 
-renderTerm : CategoryDefinition -> Html Msg
-renderTerm category =
-    dt [] [ button [ onClick (ToggleCategory category.title) ] [ text (t category.title) ] ]
+renderTerm : Model -> CategoryDefinition -> Html Msg
+renderTerm model category =
```

Comment:
Rather than requiring the model here - we could have a type (or just a bool) for isExpanded.